NovelRT.Sdk/TemplateFiles/conanfile.py

```diff
@@ -1,27 +1,19 @@
 from conans import ConanFile, CMake
-import os
 
 class ###PROJECT_NAME###Conan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake_find_package", "cmake_paths"
+    generators = "cmake_paths", "cmake_find_package"
     requires = [
-        ("NovelRT/###NOVELRT_VERSION###")]
-    default_options = {
-        "NovelRT:documentation": False,
-        "NovelRT:buildtests":False,
-        "NovelRT:buildsamples":False,
-        "NovelRT:packagemode":True
-    }
+        ("novelrt/###NOVELRT_VERSION###")
+    ]
 
     def build(self):
         cmake = CMake(self)
         cmake.configure()
         cmake.build()
 
     def imports(self):
-        self.copy("*.dll", dst="bin", src="bin")
+        if self.settings.os == "Windows":
+            self.copy("*.dll", "../deps", "bin")
+        self.copy("*.spv", "../Resources/Shaders", "bin/Resources/Shaders")

```

Comment:
Where do these end up on configure?

Reply:
They go to the root folder instead of the build folder (i.e `###PROJECT_NAME###/Resources`). This was done to keep from fragmenting the resources folder primarily, but it also gives a consistent spot to pull the dependencies from.

It should also be stated that this overwrites by default, so if something in the package changes (i.e version upgrade), the next run of `conan install` will copy newest dependencies over.